src/lib/pipeline_configuration.py

```diff
@@ -43,506 +63,780 @@ class PipelineConfiguration(object):
+                       ),
+                       CodingConfiguration(
+                           coding_mode=CodingModes.MULTIPLE,
+                           code_scheme=CodeSchemes.YOUNG_PEOPLE_ISSUES,
+                           folding_mode=FoldingModes.MATRIX,
+                           coded_field="young_people_issues_coded",
+                           analysis_file_key="young_people_issues_",
+                       )
+                   ],
                    ws_code=CodeSchemes.WS_CORRECT_DATASET.get_code_with_match_value("young_people_issues"),
-                   cleaner=None,
-                   code_scheme=CodeSchemes.YOUNG_PEOPLE_ISSUES,
-                   binary_code_scheme=CodeSchemes.YOUNG_PEOPLE_ISSUES_YES_NO_AMB,
-                   binary_coded_field="young_people_issues_yes_no_amb_coded",
-                   binary_analysis_file_key="young_people_issues_yes_no_amb"),
+                   raw_field_folding_mode=FoldingModes.CONCATENATE),
 
         CodingPlan(raw_field="decisions_minority_clan_raw",
-                   coded_field="decisions_minority_clan_coded",
                    time_field="sent_on",
                    coda_filename="decisions_minority_clan.json",
                    icr_filename="decisions_minority_clan.csv",
                    run_id_field="decisions_minority_clan_run_id",
-                   analysis_file_key="decisions_minority_clan_",
+                   coding_configurations=[
+                       CodingConfiguration(
+                           coding_mode=CodingModes.SINGLE,
+                           code_scheme=CodeSchemes.DECISIONS_MINORITY_CLAN_YES_NO_AMB,
+                           folding_mode=FoldingModes.YES_NO_AMB,
+                           coded_field="decisions_minority_clan_yes_no_amb_coded",
+                           analysis_file_key="decisions_minority_clan_yes_no_amb",
+                       ),
+                       CodingConfiguration(
+                           coding_mode=CodingModes.MULTIPLE,
+                           code_scheme=CodeSchemes.DECISIONS_MINORITY_CLAN,
+                           folding_mode=FoldingModes.MATRIX,
+                           coded_field="decisions_minority_clan_coded",
+                           analysis_file_key="decisions_minority_clan_",
+                       )
+                   ],
                    ws_code=CodeSchemes.WS_CORRECT_DATASET.get_code_with_match_value("decisions_minority_clan"),
-                   cleaner=None,
-                   code_scheme=CodeSchemes.DECISIONS_MINORITY_CLAN,
-                   binary_code_scheme=CodeSchemes.DECISIONS_MINORITY_CLAN_YES_NO_AMB,
-                   binary_coded_field="decisions_minority_clan_yes_no_amb_coded",
-                   binary_analysis_file_key="decisions_minority_clan_yes_no_amb")
+                   raw_field_folding_mode=FoldingModes.CONCATENATE),
     ]
 
     LOCATION_CODING_PLANS = [
```

Comment:
Correct me if I'm wrong, but I don't _think_ there is any need to separate the LOCATION plans from the demogs anymore?

Reply:
updated thanks.